l10n_eu_oss_oca/wizard/l10n_eu_oss_wizard.py

```diff
@@ -0,0 +1,234 @@
+# Copyright 2021 Valentin Vinagre <valentin.vinagre@sygel.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, fields, models
+from odoo.exceptions import ValidationError
+
+
+class L10nEuOssWizard(models.TransientModel):
+    _name = "l10n.eu.oss.wizard"
+    _description = "l10n.eu.oss.wizard"
+
+    def _get_default_company_id(self):
+        return self.env.company.id
+
+    def _get_eu_res_country_group(self):
+        eu_group = self.env.ref("base.europe", raise_if_not_found=False)
+        if not eu_group:
+            raise ValidationError(
+                _(
+                    "The Europe country group cannot be found. "
+                    "Please update the base module."
+                )
+            )
+        return eu_group
+
+    def _default_fiscal_position_id(self):
+        user = self.env.user
+        eu_country_group = self._get_eu_res_country_group()
+        return self.env["account.fiscal.position"].search(
+            [
+                ("company_id", "=", user.company_id.id),
+                ("vat_required", "=", True),
+                ("country_group_id", "=", eu_country_group.id),
+            ],
+            limit=1,
+        )
+
+    def _default_done_country_ids(self):
+        user = self.env.user
+        eu_country_group = self._get_eu_res_country_group()
+        return (
+            eu_country_group.country_ids
+            - self._default_todo_country_ids()
+            - user.company_id.country_id
+        )
+
+    def _default_todo_country_ids(self):
+        user = self.env.user
+        eu_country_group = self._get_eu_res_country_group()
+        eu_fiscal = self.env["account.fiscal.position"].search(
```

Comment:
When an OSS fiscal position is created using the wizard, the oss_oca boolean field is set to True. Is there any reason why that field is not used to search for the already created OSS fiscal positions? Wouldn't it be more clear? Thank you.

Reply:
@manuelregidor Yes we can, but here we search the todo_country with country & position type too, That's why.